src/powerbi-visual/README.md

```diff
@@ -76,6 +76,26 @@ You'll need to properly set up the certificate in order to be able to use the ho
 
 > Hot Reload will only work on PowerBI Web (**not** on Desktop).
 
+### Local dev guide (for powerbi-visual)
+
+1. Cd into `./src/powerbi-visual`
+1. Run `npm install`
+1. To ensure proper SSL cert usage
+   1. Ensure [mkcert](https://github.com/FiloSottile/mkcert) is installed
+   1. Run `npm run generate-certs`
```

Comment:
if you follow these instructions you should have a properly working HTTPS dev server without having to do weird hacks like "open http://localhost:8080/assets first and mark it as safe"